meta-lmp-base/conf/distro/include/lmp.inc

```diff
@@ -88,6 +88,10 @@ IMAGE_FSTYPES += "wic wic.gz wic.bmap"
 IMAGE_FSTYPES += "${@bb.utils.contains('DISTRO_FEATURES', 'sota', 'ota-ext4.gz', ' ', d)}"
 IMAGE_FSTYPES:remove = "tar.gz tar.xz tar.bz2 wic.xz wic.bz2"
 
+# For compatible with yocto < scarthgap
+WKS_FILE_DEPENDS:remove = "bmaptool-native"
+WKS_FILE_DEPENDS += "bmap-tools-native"
```

Comment:
This will break our main-next that tracks oe-core master, to make meta-freescale compatible with kirk they should create a `meta-freescale/recipes-support/bmaptool/bmaptool_git.bbappend` to provide the version available on oe-core kirk.

I believe something like the following will work but need to check locally:
```
# For compatibility with layers before scarthgap
PROVIDES += "bmaptools"

# For compatibility with layers before scarthgap
RPROVIDES:${PN} = "bmaptools"
RREPLACES:${PN} = "bmaptools"
RCONFLICTS:${PN} = "bmaptools"
```
I can prepare a PR for meta-freescale with my results if needed

Reply:
The above bbappend example doesn't work but I have another solution to fix this issue and I already validated in master and kirk

Reply:
@quaresmajose let's see, if your solution to meta-freescale passes qcom validation well and fast, it's ok. If not, this current solution is still ok to me as we just can drop the fix before switching to main-next.

Reply:
@quaresmajose yeap, confirm, your fix to meta-freescale works well. I dropped my fix.

Reply:
yeah the one on meta-freescale also helps other folks using master with oe-core kirkstone